### ISSUE TITLE
How to finetune on the "VideoLLaMA2-7B" instead of "VideoLLaMA2-7B-Base"?

### ISSUE THREAD

**Issue by Zeqing-Wang** [DAMO-NLP-SG/VideoLLaMA2#51]
Thanks for your great work! As far as I understand, the script under the "custom" folder is used to fine-tune the base model, without the fine-tuning on the instruct-tuning datasets. What should I do to continue to finetune the model on my own dataset based on the model "VideoLLaMA2-7B"?

**Comment by Zeqing-Wang** [comment]
Referring to the codes of other projects and my understanding, I directly deleted '--pretrain_mm_mlp_adapter' and replaced model_name_or_path with 'VideoLLaMA2-7B'. But I'm not sure if this is correct for VideoLLaMA2.

**Comment by lixin4ever** [comment]
Hi, thanks for your interest.

According to this [line](https://github.com/DAMO-NLP-SG/VideoLLaMA2/blob/08d91fcf2a55d5f1dc71d9db2629512c660e4db6/videollama2/train.py#L867), you can specify --output_dir with your local ${OUTPUT_DIR} and place the weights of VideoLLaMA2-7B to ${OUTPUT_DIR}/checkpoint-0 (just like you resume training from VideoLLaMA2-7B). This is the easiest way I can come up with to achieve what you said without any code change but you better check if the learning rate scheduler works as expected.

**Comment by clownrat6** [comment]
> Referring to the codes of other projects and my understanding, I directly deleted '--pretrain_mm_mlp_adapter' and replaced model_name_or_path with 'VideoLLaMA2-7B'. But I'm not sure if this is correct for VideoLLaMA2.

This issue provides continue-finetuning scripts: https://github.com/DAMO-NLP-SG/VideoLLaMA2/issues/40#issuecomment-2216328392
